boards/arm/nrf9161dk_nrf9161/nrf9161dk_nrf9161.yaml

```diff
@@ -6,7 +6,7 @@ toolchain:
   - gnuarmemb
   - xtools
   - zephyr
-ram: 88
+ram: 256
```

Comment:
I believe this should not be changed. `zephyr,sram` points to `sram0_s` and the size of this node is `DT_SIZE_K(88)`, so that's the RAM size available for samples and tests.